File: DAWES/Primer Trimestre/Actividad evaluable 1/repository/Usuarios/UsuarioRepositoryRAM.php
<?php

include_once "../../clases/Usuario.php";
include_once "UsuarioRepository.php";

class UsuarioRepositoryRAM implements UsuarioRepository {

    private $arrayUsuarios;

    public function __construct()
    {
        $this->arrayUsuarios = [];
    }

    public function guardarUsuario($usuario)
    {
        array_push($usuario);
    }

    public function getUsuarios()
    {
        return $this->arrayUsuarios;
    }
}

File: DAWES/Primer Trimestre/Actividad 1 - Evaluable/Actividad-1-Evaluable/src/main/java/com/example/demo/repository/Usuarios/UsuarioRepositoryRAM.java
package com.example.demo.repository.Usuarios;

import com.example.demo.clases.Usuario;

import java.util.ArrayList;
import java.util.List;

public class UsuarioRepositoryRAM implements UsuarioRepository{

    private ArrayList<Usuario> usuarios;

    public UsuarioRepositoryRAM() {

        this.usuarios = new ArrayList<>();
    }

    @Override
    public List<Usuario> getUsuarios() {
        return this.usuarios;
    }

    @Override
    public void guardarUsuario(Usuario usuario){
        this.usuarios.add(usuario);
    }
}
